two-tier-app-migration-to-containers-on-Azure/nested/iis.json

```diff
@@ -0,0 +1,93 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineName": {
+      "type": "string",
+      "minLength": 1
+    },
+    "sqlAuthenticationPassword": {
+         "type": "securestring",
+         "metadata": {
+             "description": "Password for the database user dbuser."
+                }
+       },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/SpektraSystems/2-Tier-app-migration-to-containers-on-Azure/master/"
+    },
+    "_artifactsLocationSasToken": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+      },
+      "defaultValue": ""
+    }
+  },
+  "variables": {
+    "vmExtensionName": "WebDeploy",
+    "nestedCustomTemplateFileName": "webDeploy.json",
+    "DSCFolder": "DSC/",
+    "nestedFolder": "nested/",
+    "dscExtensionFileName": "ContosoWebsite.zip",

```

Comment:
still need to include the source PS1 files

Reply:
I have copied this DSC extension from [here](https://github.com/Azure/azure-quickstart-templates/tree/master/dsc-extension-iis-server-windows-vm) to deploy IIS on top of SQL VM. 
I am not able to understand what do you mean by Source PS1 files.

Reply:
Inside that zip file there is a PS1 file.  Place that file into a folder named "dsc" in this sample